test/e2e/servicebindingrequest_test.go

```diff
@@ -511,3 +561,53 @@ func serviceBindingRequestTest(
 	_, err = assertDeploymentEnvFrom(todoCtx, f, deploymentNamespacedName, sbrName)
 	require.Error(t, err, "expect deployment not to be carrying envFrom directive")
 }
+
+// CreateServiceBindingRequest implements end-to-end step for creating a Service Binding Request to bind the Backing Service and the Application
+func CreateServiceBindingRequest(
```

Comment:
In the other end-to-end file we also have an `CreateSBR` method, could them share the same?

Reply:
Now we are using common `CreateSBR` method. :)